Gruntfile.js

```diff
@@ -147,6 +147,13 @@ module.exports = function(grunt) {
   var configJSON = {
     pkg: grunt.file.readJSON("package.json"),
     bump: bumpJSON,
+    umd: {
+      all: {
+        src: "plottable.js", 
+        objectToExport: "Plottable",
+        amdModuleId: "Plottable",
```

Comment:
I think we need to set `objectToExport`, since an object (named "`Plottable`") is being exported.

`amdModuleId` seems like it shouldn't be left out since I don't think we want the module to be anonymous.